README.md

```diff
@@ -1,4 +1,5 @@
 # Core Vanilla JavaScript
+Grey Toucan Please see solo study Guide for my work.
 
 This repo contains resources, challenges and plans for learning the basics of Javascripts.
```

Comment:
does this project have any exercises to complete? Based on my review it looks like you were working on the tutorials and did not get to the exercises. It is also possible the empty branch was not setup properly https://playbook.learnersguild.org/Learning_Guide/Review.html